server/src/services/largeLanguageModel.ts

```diff
@@ -12,17 +11,17 @@ const defaultClient = new OpenAI({
 })
 
 // Makes a request to OpenAI's API
-export async function sendGPTQuery(prompt: string, client: OpenAI = defaultClient) {
-  const response = await client.chat.completions
-    .create({
-      model: MODEL,
-      messages: [{ role: 'user', content: prompt }],
-    })
-    .then((response: ChatCompletion) => {
-      return response
-    })
-
+export async function sendGPTQuery(
+  prompt: string,
+  client: OpenAI = defaultClient,
+): Promise<string> {
+  const response = await client.chat.completions.create({
+    model: MODEL,
+    messages: [{ role: 'user', content: prompt }],
+  })
+  if (!response.choices[0].message.content) {
```

Comment:
Voisko tässä käyttää mielummin try/catch?

Reply:
Tein tän näin koska halusin, että ton funktion palauttama tyyppi on on string eikä ``string | null``, koska me ei oletettavasti tehdä mitään null vastauksella. Toi content on tyypitetty näin``(property) ChatCompletionMessage.content: string | null``.  Content saattaa siis olla``null`` vaikka ite toi API kutsu ei heittäis mitään erroria.